apps/test/unit/code-studio/pd/workshop_dashboard/WorkshopLinksTest.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 
 import WorkshopLinks from '@cdo/apps/code-studio/pd/workshop_dashboard/WorkshopLinks';
 
-describe('EnrollInWorkshop', () => {
+describe('WorkshopLinks', () => {
```

Comment:
I realized I forgot to update the name on this file when I created it a couple weeks ago 😵